### ISSUE TITLE
About pretrained model 

### ISSUE THREAD

**Issue by chartgod** [yuhongtian17/Spatial-Transform-Decoupling#9]
Where is the source code to predict using the pre-trained model?

HRSC2016
STD with Oriented RCNN and ViT-B

**Comment by yuhongtian17** [comment]
Do you mean just loading the ViT pretrained model and making a test without any training epochs?

**Comment by chartgod** [comment]
Yes, that’s right. We would like to test the test dataset by loading the pre-trained model.

**Comment by yuhongtian17** [comment]
Method 1: use torch.load() to load the pretrained model, change the key name (seem to add "model.backbone."?) and use torch.save() to re-save the model.

Method 2: cut down the number of train dataset to only 1 picture, change the learning rate to 0 and train for 1 epoch. Then you can use epoch_1.pth for your need.

**Comment by chartgod** [comment]
Thx
